### PR TITLE
Update robustus.py

### PR DESCRIPTION
**Pull request by oliverbmartin**
fix debug message with active branch error

@olegsinyavskiy please have a look and merge.

### CODE REVIEW

File: robustus/robustus.py [hunk 1]
```diff
@@ -553,7 +553,7 @@ def install(self, args):
                 active_branch = re.search('origin/\w*', msg).group(0)
                 # http://stackoverflow.com/questions/6657690/python-getoutput-equivalent-in-subprocess
             except Exception as err:
-                active_branch = '<Could not find active branch - %s: %s>' % (err.__class__.__name__, err.message)
+                active_branch = '<Could not find active branch - %s: %s>. Branch out of date. Try running git pull' % (err.__class__.__name__, err.message)
```
Comment (olegsinyavskiy) [inline robustus/robustus.py]: Please change to "Branch might be out of date". I just think there might be other reasons to get this error and people will be confused again.

Comment (oliverbmartin) [inline robustus/robustus.py]: I'm getting the same error when I switch branches (ie to ppm_utils) and run source use_repo.sh. I tried git pull origin ppm_utils as well as git pull origin master while in the ppm_utils branch, but that doesn't seem to solve the problem. I'll look into it a bit more and make the error message a little more explicit. @bc-jmoork, fyi.

**Comment by oliverbmartin** [comment]
Turns out that you need to synchronize in both directions for this to work. I had made some local changes to the branch that I was working on and hadn't pushed them to origin. I tried to make it clear that the user should try both git pull and git push.

**Comment by olegsinyavskiy** [comment]
@oliverbmartin Nice! Thanks!

**Pull request merged**